homeassistant/package_constraints.txt

```diff
@@ -32,6 +32,3 @@ enum34==1000000000.0.0
 
 # This is a old unmaintained library and is replaced with pycryptodome
 pycrypto==1000000000.0.0
-
-# Contains code to modify Home Assistant to work around our rules
-python-systemair-savecair==1000000000.0.0
```

Comment:
Do you known why you lands on this list?

Reply:
This was discussed in a previous PR, but to recall. I kept two repository, to be private for my own HA setup, and one public that had the limitations of HA components. The remotes where mixed and the publish script I used published code in violation. Therefore on the list. The pypi package was however removed and is updated with the code on GitHub